aidants_connect_web/migrations/0049_auto_20210321_2154.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.1 on 2021-03-21 20:54
+
+from django.db import migrations
+
+orga_types = [
+    "France Services/MSAP",
+    "CCAS",
+    "Centres sociaux",
+    "Sécrétariats de mairie",
+    "Maisons départementales des solidarités",
+    "Médiathèque",
+    "Autre guichet d’accueil de service public de proximité",
+    "Guichet d’accueil d’opérateur de service public (CAF, Pôle Emploi, etc.",
```

Comment:
Il manque une petit parenthèse fermante, là, nan ? :thinking: 

```suggestion
    "Guichet d’accueil d’opérateur de service public (CAF, Pôle Emploi, etc.)",
```